docs/distributing-policies/publish-policy-to-artifact-hub.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: "Publish to Artifact Hub"
+title: ""
+---
+
+# Publish Policy to Artifact Hub
+
+[Artifact Hub](https://artifacthub.io/) is a website where users can find,
+install and publish packages and configurations for CNCF projects.
+
+Kubewarden policies can be published on Artifact Hub and made
+discoverable to the wide audience of CNCF users.
+
+:::note
+Artifact Hub is content aggregator, it doesn't actually host the
+artifacts that are published on it.
+
+Kubewarden Policies need to be physically hosted by a container
```

Comment:
So this is a general requirement for Artifact Hub and I think should be stated as such.
```suggestion
Artifact Hub requires you to physically host container image repositories on a container
```